index.html

```diff
@@ -713,6 +713,8 @@ <h2 class='informative'>Vary</h2>
         <section id="persistence-fixity-preamble" class="informative">
           <h2>Extension of <code>Expect</code> header</h2>
           <p>
+            Persistence fixity is verified by comparison of stored fixity results with newly
+            computed fixity results using <code>HEAD</code> or <code>GET</code> requests to <a>LDP-NR</a>s.
```

Comment:
My thought was that is has to be stored somewhere, in the client or the server, but I could go with `previous` --- will change